src/PKSim.Core/Services/PKSimParameterStartValuesCreator.cs

```diff
@@ -75,40 +58,6 @@ private bool isChangedSimulationParameter(IParameter parameter)
                 && parameter.ValueDiffersFromDefault();
       }
 
-      private void updateMoleculeParametersValues(IndividualMolecule molecule, Individual individual, Simulation simulation)
-      {
-         var allMoleculeParameters = individual.AllMoleculeParametersFor(molecule);
-         var modelConfiguration = simulation.ModelConfiguration;
-         allMoleculeParameters.Each(p => setParameter(p, modelConfiguration.ModelName == CoreConstants.Model.FOUR_COMP));
-      }
-
-      private void setParameter(IParameter parameter, bool isSmallMolecule)
-      {
-         //We do not generate start values for endosome parameters when dealing with a small molecule model
-         if (parameter.HasAncestorNamed(CoreConstants.Compartment.ENDOSOME) && isSmallMolecule)
-            return;
-
-         //We do not generate value for this parameter by default. Exit
-         var parameterStartValue = getStartValueFor(parameter);
-         if (parameterStartValue == null)
-            return;
-
-         if (parameter.Formula.IsExplicit())
-         {
-            var formula = _formulaFactory.RateFor(EXPRESSION_PARAMETERS, parameter.Formula.Name, _defaultStartValues.FormulaCache);
-            parameterStartValue.Formula = formula;
-            //There is a formula, make sure we use it. We set this flag to false to ensure that the formula will not be replaced with a constant formula
-            parameterStartValue.OverrideFormulaWithValue = false;
-         }
-
-         if (parameter.IsConstantParameter())
-         {
-            parameterStartValue.StartValue = parameter.Value;
-            //we reset the formula to null to ensure that the start value will be used when constructing the simulation
-            parameterStartValue.Formula = null;
```

Comment:
@Yuri05 Interestingly enough, constant parameter seem to take over formula anyways as I found out with @rwmcintosh last time. I am a a bit surprised by this behavior